jest-dev-server.config.js

```diff
@@ -1,14 +1,11 @@
 const { ports } = require('./config/index.js')
 
-const PORT = process.env.PORT || ports.test
-
 /**
  * @type {import('jest-dev-server').JestDevServerOptions}
  */
 module.exports = {
-  // Ensure PORT works (on Windows) + SIGINT/SIGTERM signal events
-  command: `cross-env-shell PORT=${PORT} node app/start.js`,
-  port: PORT,
+  command: 'npm start --workspace app',
```

Comment:
We confident we don't need `cross-env` to handle signals still, as per the comment above it?

Reply:
Sorry it was badly worded by me originally!

It was to explain [`cross-env` vs `cross-env-shell`](https://github.com/kentcdodds/cross-env#cross-env-vs-cross-env-shell)

>On Windows you need to use cross-env-shell, if you want to handle [signal events](https://nodejs.org/api/process.html#process_signal_events) inside of your program. A common case for that is when you want to capture a SIGINT event invoked by pressing Ctrl + C on the command-line interface.

Neither of them are needed if we drop `PORT`